nemo/export/trt_llm/converter/model_to_trt_llm_ckpt.py

```diff
@@ -118,37 +135,28 @@ def convert_model_to_trt_llm_ckpt(
 
     has_position_embedding = get_layer_name("position_embedding", prefix) in model_state_dict
     has_lm_head = get_layer_name("output_layer", prefix) in model_state_dict
```

Comment:
Why are some of the values removed?

Reply:
[I believe these were refactored into here](https://github.com/NVIDIA/NeMo/pull/10096/files#diff-4a09132034fd9a231315e092fa057c1ad7a7db379985c97d776a8dd1ea995b25R83)

Reply:
Indeed, I've moved the common parts to a single place